### PR TITLE
Using Metamask with Celo

### PR DESCRIPTION
**Pull request by joenyzio**
Adding back doc updates lost in revert. Using MetaMask with Celo updates to clarify warnings and tutorial language.

### CODE REVIEW

File: packages/docs/getting-started/using-metamask-with-celo/README.md [hunk 1]
```diff
@@ -1,32 +1,44 @@
...
+While gas on Celo can usually be paid in [many different currencies](https://docs.celo.org/celo-codebase/protocol/transactions/erc20-transaction-fees), when using MetaMask, gas fees will automatically be paid in CELO. This is because MetaMask will be using the [Ethereum-compatible Celo transaction format](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0035.md), which doesn't include the `feecurrency` field.
 
-### Logo
+## **Incorrect Logo**
 
-* The MetaMask UI might show the ETH logo in places where it's meant to show the CELO logo or no logo at all.
+In some cases, the MetaMask UI may display the Ethereum logo in places where it should display a CELO logo or no logo at all.
+
+{% hint style="warning" %}
+MetaMask is primarily used for interacting with the Ethereum blockchain and does not natively support Celo compatibility. Celo recommends only sophisticated users and developers use it. Alternatively, you may choose a Celo native wallet [here](https://docs.celo.org/getting-started/wallets).
```
Comment (critesjosh) [inline packages/docs/getting-started/using-metamask-with-celo/README.md]: Are we still recommending that only sophisticated users and developers use Metamask, or are we now recommending it for everyone (considering pool together only supports metamask)?

File: packages/docs/getting-started/using-metamask-with-celo/README.md [hunk 2]
```diff
@@ -1,32 +1,44 @@
...
+
+Since Celo network's [Donut Hardfork](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0027.md), activated on Mainnet on May 19th, 2021, the protocol now supports [Ethereum-compatible transactions](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0035.md). This means that users may use MetaMask to interact with the Celo blockchain and dApp developers can more easily port Ethereum dApps to the Celo blockchain.
+
+# **How to use MetaMask with Celo**
+
+### **For end users:**
 
-With the Celo network's [Donut Hardfork](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0027.md), which was activated on Mainnet on May 19th, 2021, the protocol now supports [Ethereum-compatible transactions](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0035.md). This means that users are now able to use MetaMask to interact with the Celo blockchain and dApp developers can more easily port over Ethereum dApps onto Celo. The following guide aims to detail step-by-step how to do that.
+* [Configure a MetaMask Desktop or Web Wallet to Work with Celo](https://docs.celo.org/getting-started/wallets/using-metamask-with-celo/manual-setup)
```
Comment (critesjosh) [inline packages/docs/getting-started/using-metamask-with-celo/README.md]: There are a bunch of links directly to https://docs.celo.org throughout this doc. Relative links are preferred so they can be caught by link checkers

File: packages/docs/getting-started/using-metamask-with-celo/README.md (hunk 1 above)
Comment (ericnakagawa) [reply packages/docs/getting-started/using-metamask-with-celo/README.md]: This should be moved to bottom.

We want to diminish severity of the warning and let people choose how they wish to interact. PT needs MM, so yes recommending it as a viable option versus warning to stay away.

Comment (joenyzio) [reply packages/docs/getting-started/using-metamask-with-celo/README.md]: Kept at the bottom with lighter language than before

File: packages/docs/getting-started/using-metamask-with-celo/README.md (hunk 2 above)
Comment (joenyzio) [reply packages/docs/getting-started/using-metamask-with-celo/README.md]: Added relative links for docs

File: packages/docs/getting-started/using-metamask-with-celo/README.md [hunk 3]
```diff
@@ -1,32 +1,44 @@
...
+
+# **How to use MetaMask with Celo**
+
+### **For end users:**
 
-With the Celo network's [Donut Hardfork](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0027.md), which was activated on Mainnet on May 19th, 2021, the protocol now supports [Ethereum-compatible transactions](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0035.md). This means that users are now able to use MetaMask to interact with the Celo blockchain and dApp developers can more easily port over Ethereum dApps onto Celo. The following guide aims to detail step-by-step how to do that.
+* [Configure a MetaMask Desktop or Web Wallet to Work with Celo](manual-setup.md)
+* [Setup a Ledger to Work with MetaMask to Work With Celo](packages/docs/getting-started/using-metamask-with-celo/using-a-ledger-with-metamask.md)
```
Comment (critesjosh) [inline packages/docs/getting-started/using-metamask-with-celo/README.md]: these relative links are broken. the path is relative to the current file, so `packages/docs/getting-started/using-metamask-with-celo/using-a-ledger-with-metamask.md` should be `using-a-ledger-with-metamask.md`. You can check to make sure links are working properly by previewing the file and clicking the links. Broken links will give you a 404.

Comment (joenyzio) [reply packages/docs/getting-started/using-metamask-with-celo/README.md]: Sorry about that updated links

**Review by critesjosh — APPROVED** [review]
looks good!

**Pull request merged**